README.md

```diff
@@ -15,6 +15,13 @@ English | [简体中文](README_zh-CN.md)
 
 ## ✨ MindSearch: Mimicking Human Minds Elicits Deep AI Searcher
 
+## 🚀 New Features in This Forked Version
+
+This fork of MindSearch has introduced the following new features:
+
+- **Environment Variables Configuration**: The application now uses a `.env` file to manage environment variables, making it easier to configure and switch between different environments.
+- **Backend Usage Example**: Added `backend_example.py` to demonstrate how to interact with the backend directly. This is particularly useful for users who prefer not to use the frontend.
```

Comment:
maybe not need

Reply:
Thank you for your feedback! I removed this section.